README.md

```diff
@@ -43,7 +43,7 @@ You might be interested in:
   [write](https://zulip.readthedocs.io/en/latest/overview/contributing.html#zulip-outreach)
   for the Zulip blog, or
   [give us feedback](https://zulip.readthedocs.io/en/latest/overview/contributing.html#user-feedback). We
-  would love to hear from you, even if you're just trying the product out.
+  would love to hear from you, even if you're trying the product out.
```

Comment:
I think the "just" was preferable without this change.

Reply:
Hi,
the grammerly site pointed out that the above mentioned sentence is wordy and removing "just" solved that problem that's why I changed it. 😄